keyboards/xbows/knight_plus/knight_plus.c

```diff
@@ -0,0 +1,154 @@
+  {1, C6_10, C5_10, C4_10},   // L68
+  {1, C6_11, C5_11, C4_11},   // L69
+  {1, C6_14, C5_14, C4_14},   // L70
+  {1, C6_15, C5_15, C4_15},   // L71
+  {1, C6_16, C5_16, C4_16},   // L72
+  {2, C9_6, C8_6, C7_6},      // L73
+
+  {0, C9_9, C8_9, C7_9},      // L74
+  {0, C9_10, C8_10, C7_10},   // L75
+  {0, C9_11, C8_11, C7_11},   // L76
+  {0, C9_12, C8_12, C7_12},   // L77
+  {0, C9_13, C8_13, C7_13},   // L78
+  {0, C9_14, C8_14, C7_14},   // L79
+  {1, C9_11, C8_11, C7_11},   // L80
+  {1, C9_12, C8_12, C7_12},   // L81
+  {1, C9_13, C8_13, C7_13},   // L82
+  {1, C9_14, C8_14, C7_14},   // L83
+  {2, C1_6, C2_6, C3_6},      // L84
+  {2, C1_7, C2_7, C3_7},      // L85
+  {2, C1_8, C2_8, C3_8},      // L86
+ };
+
+ led_config_t g_led_config = { {
+     {  0,      1,      2,      3,      4,      5,      6,      7,      8,      9,      10,     11,     12,     13,     14     },
+     {  15,     16,     17,     18,     19,     20,     NO_LED, 21,     22,     23,     24,     25,     26,     27,     28     },
+     {  29,     30,     31,     32,     33,     34,     35,     36,     37,     38,     39,     40,     41,     42,     43     },
+     {  44,     45,     46,     47,     48,     49,     50,     51,     52,     53,     54,     55,     56,     57,     58     },
+     {  59,     60,     61,     62,     63,     64,     65,     66,     67,     68,     69,     70,     71,     72,     NO_LED },
+     {  73,     74,     75,     NO_LED, 76,     77,     78,     79,     NO_LED, 80,     81,     82,     83,     84,     85     }
+ }, {
+ 		{0,0},  {16,0},  {30,0},  {43,0},  {56,0},  {72,0},  {85,0},   {99,0},   {112,0},  {128,0},  {141,0},  {154,0},  {167,0},  {190,0},  {216,0},
+ 		{0,13}, {16,13}, {32,12}, {49,14}, {66,16}, {82,20},           {102,20}, {118,16}, {134,14}, {150,12}, {167,13}, {184,13}, {197,13}, {216,13},
+ 		{0,26}, {15,26}, {31,23}, {47,24}, {63,28}, {78,30}, {105,30}, {120,28}, {136,24}, {152,23}, {168,26}, {184,26}, {197,26}, {210,26}, {224,26},
+ 		{0,37}, {15,37}, {30,33}, {45,35}, {60,37}, {75,40}, {92,43},  {108,40}, {124,37}, {138,35}, {153,33}, {168,37}, {184,37}, {203,37}, {224,37},
+ 		{0,48}, {14,48}, {29,44}, {43,46}, {57,48}, {72,50}, {92,53},  {112,50}, {126,48}, {141,46}, {155,44}, {169,48}, {184,48}, {210,48},
+ 		{0,58}, {14,58}, {34,58},          {61,61}, {83,64}, {100,64}, {122,61},           {149,58}, {170,58}, {184,58}, {197,58}, {210,58}, {224,58}
+ }, {
+     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+     4, 4, 4, 4, 4, 4,    4, 4, 4, 4, 4, 4, 4, 4,
+     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+     4, 4, 4,    4, 4, 4, 4,    4, 4, 4, 1, 1, 1
+ } };
+
+
+void suspend_power_down_kb(void) {
+    rgb_matrix_set_suspend_state(true);
+    suspend_power_down_user();
+}
+
+void suspend_wakeup_init_kb(void) {
+    rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+}
```

Comment:
```suggestion
```

Reply:
Can you tell me why you want to delete these lines of code?